vault/dendron.tutorial.linking-notes.md

```diff
@@ -2,43 +2,59 @@
 id: 9MZBqhrijEM4QpZRa5t08
 title: Linking Notes
 desc: Linking Notes
-updated: 1637551418396
+updated: 1639428297065
 created: 1625563999532
 nav_order: 2
 ---
 
 ### Links
 
-Dendron supports links between your notes, which can help you organically connect your thoughts and build up a powerful and **personal** knowledge graph.
+Dendron supports links between your notes, which can help you organically connect your thoughts and build up a powerful and personal knowledge graph.
 
-> 🌱 To create a link, just start typing [[ and Dendron will show you a list of notes in your workspace that you can link to. Try it in the editor pane now.
+> 🌱 **ACTION:** To create a link, start typing `[[` and Dendron will show you a list of notes in your workspace that you can link to. Try it in the editor view pane now.
```

Comment:
for action, lets give user something concrete. eg. link to `recipes.vegetarian`

- NOTE: user won't have access to dendron wiki in the tutorial, we're exporting the tutorial hiearchy which means note references and wikilinks to the wiki will be inlined and URLs will be added to wiki links